src/index.ts

```diff
@@ -12,7 +12,7 @@ import { WaxSigningApi } from "./WaxSigningApi";
 const PROOF_WAX = 1;
 const PROOF_USER = 2;
 export class WaxJS {
-  public readonly rpc: JsonRpc;
+  public rpc: JsonRpc;
```

Comment:
why we rm readonly on this property? 
@sonnguyenwax so

Reply:
@thien-hoang-opskins if it stay at read only, we cannot assign it back to new rpc when doing action setChainName
because readonly only allowed to set once at construct